### PR TITLE
DOCSP-36191: Update Log pages with new Event Handler log type

### PR DESCRIPTION
**Pull request by cbullinger**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-36191

- [View Application Logs](https://preview-mongodbcbullinger.gatsbyjs.io/atlas-app-services/docsp-36191-eventbridge-logs/activity/view-logs/): Add `trigger_error_handler` log type to CLI section
- [Forward Logs to a Service](https://preview-mongodbcbullinger.gatsbyjs.io/atlas-app-services/docsp-36191-eventbridge-logs/activity/forward-logs/#2.-choose-which-logs-to-forward): Add `trigger_error_handler` log type to list of supported CLI types
- [App Log Types](https://preview-mongodbcbullinger.gatsbyjs.io/atlas-app-services/docsp-36191-eventbridge-logs/logs/): Add new Trigger Error Handler Logs ref to list
- [Trigger Error Handler Logs](https://preview-mongodbcbullinger.gatsbyjs.io/atlas-app-services/docsp-36191-eventbridge-logs/logs/error-handler/): New reference page
- [Admin API: Get Logs](https://preview-mongodbcbullinger.gatsbyjs.io/atlas-app-services/docsp-36191-eventbridge-logs/admin/api/v3/#tag/logs/operation/adminGetLogs): Update endpoint with new `TRIGGER_ERROR_HANDLER` log type and add new log response schema 

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Did you tag pages appropriately?
  - genre
  - programming_language
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-realm work, if any

### Release Notes

- **Monitor App Activity**
  - View Application Logs: Update page with new `trigger_error_handler` CLI log type in support of AWS EventBridge custom error handling.
  - Forward Logs to a Service: Update list of supported CLI log types with new `trigger_error_handler` type in support of AWS EventBridge custom error handling.
  - Log Types > Trigger Error Handler Logs: New page documenting the trigger error handler log type in support of AWS EventBridge custom error handling.
- **References**
  - App Services Admin API: Update /adminGetLogs endpoint with new `TRIGGER_ERROR_HANDLER` log type schema in support of AWS EventBridge custom error handling.  

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/activity/view-logs.txt [hunk 1]
```diff
@@ -101,6 +112,7 @@ The following types are valid:
 - ``graphql``
 - ``sync``
 - ``schema``
+- ``errorhandler``
```
Comment (cbullinger) [inline source/activity/view-logs.txt:115]: Need to confirm this value once the [CLI type list](https://github.com/10gen/baas/blame/7911f2c40da976cbaa51f9d28f5f7bdd69309713/cli/commands/logs/list_inputs.go#L15-L15C35) is updated

File: source/activity/forward-logs.txt
Comment (cbullinger) [inline source/activity/forward-logs.txt:1]: @lindseymoore Need to update the screenshot for the types dropdown (/images/log-forwarding-type.png). Are you able to access that field/dropdown?

**Review by lindseymoore — APPROVED** [review]
LGTM, pending confirmation of the ``errorhandler`` flag!

File: source/activity/view-logs.txt [hunk 2]
```diff
@@ -101,6 +112,7 @@ The following types are valid:
 - ``graphql``
 - ``sync``
 - ``schema``
+- ``error_handler``
```
Comment (cbullinger) [inline source/activity/view-logs.txt:115]: need to confirm this once it's updated in CLI

Comment (tcannon91) [reply source/activity/view-logs.txt:115]: Just to close the loop a bit -- this ticket for this work will be in the next release: https://jira.mongodb.org/browse/BAAS-28612

File: source/logs/error-handler.txt [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Event handler logs are created whenever Atlas executes a custom error
```
Comment (tcannon91) [inline source/logs/error-handler.txt:23]: Should this be `Error handler logs...`? 
```suggestion
Error handler logs are created whenever Atlas executes a custom error 
```

Comment (cbullinger) [reply source/logs/error-handler.txt:23]: Derp! yes, thanks for catching

File: source/activity/view-logs.txt (hunk 2 above)
Comment (cbullinger) [reply source/activity/view-logs.txt]: updated to `trigger_error_handler` throughout

File: source/images/log-forwarding-type.png
Comment (cbullinger) [inline source/images/log-forwarding-type.png:1]: needs to be updated once available (@lindseymoore can you provide?)

**Review by tcannon91 — APPROVED** [review]
🚀

**Pull request merged**